mirror-service-x/src/main/resources/application.properties

```diff
@@ -45,4 +45,7 @@ quarkus.kafka.devservices.topic-partitions."dtrack.notification.datasource-mirro
 
 mirror.datasource.nvd.api-key=
 mirror.datasource.nvd.num-threads=4
-mirror.datasource.github.api-key=
\ No newline at end of file
+mirror.datasource.github.api-key
+## OSV mirroring
+mirror.osv.baseurl=https://osv-vulnerabilities.storage.googleapis.com
+mirror.datasource.github.base-url
```

Comment:
Are the `=`s missing for the GItHub properties?

Reply:
fixed